content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -400,18 +399,90 @@ $ curl \
 }
 ```
 
+## Show workspace
+
+This endpoint shows details for a workspace in the organization.
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /organizations/:organization/workspaces/:name |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - Specifies the organization name under which to list the workspaces. This is specified in the URL path.
+- `:name` (`string: <required>`) - Specifies the name of the workspace to show details for. This is specified in the URL path.
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/organizations/my-organization/workspaces/my-workspace-3
```

Comment:
I'd probably use the same workspace name here as well, unless there's a reason not to?

Reply:
Changing to my-workspace-1.